rnd/autogpt_server/autogpt_server/agent_executor/executor.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import time
+
+from multiprocessing import Process
+from concurrent.futures import ThreadPoolExecutor
+
+from autogpt_server.data import ExecutionQueue
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: Replace this by an actual Agent Execution.
+def __execute(id: str, data: str) -> None:
```

Comment:
Don't love dunderscore (double underscore) methods outside of classes. Also could be useful to bind this to a class sooner rather than later

Reply:
Moved to class